src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <IsPackable>true</IsPackable>
 
-    <Version>2.0.3</Version>
+    <Version>2.0.4</Version>
     <PackageReleaseNotes>Upgrade to netstandard2.1</PackageReleaseNotes>
```

Comment:
Can you please add a description of what this change is about?

Reply:
Inside the `<PackageReleaseNotes>` tag! Cheers!

Reply:
Just the <PackageReleaseNotes> tag? - sure

Reply:
Brilliant thanks